terraform/modules/core/core.tf

```diff
@@ -231,6 +231,46 @@ resource "aws_lb_listener_rule" "redirect_old_teachingjobs_http_traffic" {
   }
 }
 
+resource "aws_lb_listener_rule" "redirect_old_teachingjobs_https_traffic_with_www_subdomain" {
```

Comment:
I wonder how will this work in non-production environments, since the other uses of listener rules are controlled with boolean to be enabled only in production?

Reply:
Turns out it works as expected, the rule is created but doesn't get invoked.

Reply:
![screenshot 2019-01-21 at 17 00 36](https://user-images.githubusercontent.com/912473/51489061-1d4b7200-1d9f-11e9-96d3-d1966bcff3db.png)